logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -42,8 +41,7 @@ class UserDataSource(
     private val userApi: UserDetailsApi,
     private val idMapper: IdMapper,
     private val userMapper: UserMapper,
-    private val assetRepository: AssetRepository,
-    private val teamRepository: TeamRepository
+    private val assetRepository: AssetRepository
```

Comment:
The amount of dependencies here really made it clear that it was doing too much stuff.